sunpy/map/header_helper.py

```diff
@@ -361,3 +362,70 @@ def _set_instrument_meta(meta_wcs, instrument, telescope, observatory, wavelengt
     'CD2_2':
     'Matrix element CDi_j describing the rotation required to align solar North with the top of the image.'
 }
+
+
+def make_heliographic_header(date, observer_coordinate, shape, frame, *, projection_code="CAR"):
```

Comment:
How would you feel about this? I think it makes calling this a lot clearer if the `frame=` is there.
```suggestion
def make_heliographic_header(date, observer_coordinate, shape, *, frame, projection_code="CAR"):
```

Reply:
👍  done